scripts/west_commands/runners/nrfjprog.py

```diff
@@ -45,7 +46,7 @@ def do_add_parser(cls, parser):
                             action='store_true',
                             help='use reset instead of pinreset')
         parser.add_argument('--snr', required=False,
-                            help='serial number of board to use')
+                            help='Serial number of board to use. \'*\' matches one or more characters/digits.')
```

Comment:
Please keep this 80 column clean using a triple quoted string

```suggestion
                            help="""Serial number of board to use.
                            '*' matches one or more characters/digits.""")
```